File: countOfOccurence.py
#Find the number of occurence of a character in a string
myString = "aaannjitha"
charCount = {}
for char in myString:
    if char in charCount:
        charCount[char] += 1
    else:
        charCount[char] = 1
for char,count in charCount.items():
    print(char + " Occurs " + str(count) + " times ")
    

File: countOfOccurence.cpp
#include <iostream>
#include <set>
#include <map>
#include <string>

int main() {
    std::string myString = "aaannjitha";
    std::set<char> charSet;
    std::map<char, int> charCount;

    for (char c : myString) {
        charSet.insert(c);
    }

    for (char c : charSet) {
        int count = 0;
        for (char ch : myString) {
            if (c == ch) {
                count++;
            }
        }
        charCount[c] = count;
    }

    for (const auto& pair : charCount) {
        std::cout << pair.first << " Occurs " << pair.second << " times" << std::endl;
    }

    return 0;
}
